Packs/Bonusly/Integrations/Bonusly/Bonusly.py

```diff
@@ -0,0 +1,357 @@
+"""Bonusly Integration for Cortex XSOAR (aka DBot)
+
+This integration only has a few commands and more maybe added depending on feedback.
+
+"""
+
+import json
+
+import demistomock as demisto
+import requests
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+import dateparser
+
+# IMPORTS
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+# Client Class
+class Client(BaseClient):
+    """
+
+    This is the base client used for Bonusly http requests
+
+    """
+
+    def bonusly_test(self):
+        suffix = 'bonuses'
+        self._http_request('GET', suffix)
```

Comment:
```suggestion
        return self._http_request('GET', suffix)
```

I see your test_module is failing, I believe this is the reason